cmd/ingress-operator/http2-testserver.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	http2testserver "github.com/openshift/cluster-ingress-operator/test/http2"
+	"github.com/spf13/cobra"
+)
+
+func NewServeHTTP2TestServerCommand() *cobra.Command {
+	var command = &cobra.Command{
+		Use:   "serve-http2-testserver",
```

Comment:
```suggestion
		Use:   "serve-http2-test-server",
```
testserver should be test-server probably (not just here)?